velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -57,12 +49,14 @@ class WindowTestBase : public exec::test::OperatorTestBase {
   // including input parameters and open/close braces. e.g. rank(), ntile(5)
   void testTwoColumnOverClauses(
       const std::vector<RowVectorPtr>& input,
-      const std::string& windowFunction);
+      const std::string& windowFunction,
+      const std::string& overClause,
+      const std::optional<std::string>& frameClause = "");
 
- private:
   void testWindowFunction(
       const std::vector<RowVectorPtr>& input,
       const std::string& function,
-      const std::string& overClause);
+      const std::string& overClause,
+      const std::optional<std::string>& frameClause = "");
```

Comment:
Its confusing if the parameter is optional with a default value of empty. It would be better to remove optional and use the default value or use optional and remove the default value.

Reply:
Sure, will remove optional, thanks for the suggestion!